bin/ledger2beancount

```diff
@@ -382,143 +518,88 @@ EOT
     print "plugin \"$_\"\n" foreach @{$config->{plugins}};
 }
 
-print "\n";
-
 
 # MAIN CONVERSION LOOP
 
-while (my $l = <>) {
-    # print "D: line: ", "\n"; dump_cur_txn(); print "\n";
-
-    # handle line indentation once and for all
+my @input = <stdin>;
+while (@input) {
+    my $l = shift @input;
```

Comment:
This has changed the CLI interface of the conversion scripts. Before they could be used both as a real filter, with `<` redirection, and by passing input files via argv. Now only the former is possible, breaking the old interface. It would be nice to have it back, as most filters also works with argv parameters (e.g., `cat`)

Reply:
I had no idea this was possible.